### ISSUE TITLE
DVS Gesture preprocessing and learning

### ISSUE THREAD

**Issue by tasbolat1** [bamsumit/slayerPytorch#18]
Hello @bamsumit,

I have two questions related to experiments related to  DVS Gesture dataset:
1.
You have mentioned that you've used 30 as false postive and 180 as true positive for DVS gesture dataset. However, I can't find the info on how you processed the data. How the DVS gesture dataset was loaded to the model? is there any binning? By binning I refer to the process when the tSample is fixed or depth of the samples (5th dimension) are fixed.

2. I can't find the exact architecture for 8-layers used for learning DVS Gesture dataset. Is the any delay functions added there? What are the hidden unit sizes?

Thanks,
Tasbolat

**Comment by bamsumit** [comment]
Hi Tasbolat,
Please look at DVS Gesture Loihi example. The dataloader is the same. It does OR binning.
The architecture in the paper is `128x128x2-4a-16c5d-2a-32c3d-2a-512d-11`. Yes it does use delays.
Regards,
Sumit

**Comment by tasbolat1** [comment]
Hello @bamsumit ,
I have followed the instructions you have given and trained the model by your [architecture.](https://github.com/bamsumit/slayerPytorch/blob/master/exampleLoihi/03_IBMGesture/ibmGestureTrain.py) First, I binned the data with 1450 bins with OR. Second, for training set I chose first 23 subjects, the rest are testing. Here are my network yaml

```
params = {
    "neuron": {
        "type": "SRMALPHA",
        "theta": 10,
        "tauSr": 10.0,
        "tauRef": 1.0,
        "scaleRef": 2,
        "tauRho": 1, # pdf
        "scaleRho": 1,
    },
    "simulation": {"Ts": 1.0, "tSample": 1450, "nSample": 1},
    "training": {
        "error": {
            "type": "NumSpikes",  # "NumSpikes" or "ProbSpikes"
            "tgtSpikeRegion": {  # valid for NumSpikes and ProbSpikes
                "start": 0,
                "stop": 1450,
            },
            "tgtSpikeCount": {True: 180, False: 30},
        }
    },
}
```
However I am unable to get same result as [yours.](https://github.com/bamsumit/slayerPytorch/blob/master/exampleLoihi/03_IBMGesture/TrainedFull/accuracy.png). Here is my result:
![DVS_results_single](https://user-images.githubusercontent.com/4617666/81374092-fdc18f80-9130-11ea-9a62-9438fa67b884.png)

As you can see, the testing accuracy is far below than yours. Is there anything that I am missing?

Kind Regards,
Tasbolat
